Contribution.md

```diff
@@ -0,0 +1,78 @@
+
+Before you begin contributing, make sure you have:
+
+- Rust and Cargo installed on your system.
+- To take advantage of simd-json your system needs to be SIMD-capable
+- A GitHub account for version control and issue tracking.
+- Make sure you are using the newly released version
+- Familiarized yourself with the project by reviewing the reviewing the example in this repository `example` folder and understanding its goals.
+
+## Code of Conduct
+
+Please review our [Code of Conduct](CODE_OF_CONDUCT.md) to understand the expected behavior and conduct within the project's community. We expect respectful and professional interactions from all participants.
+
+## How to Contribute
+
+We welcome contributions in various forms:
+
+### Reporting Bugs
+
+If you discover any bugs, issues, or unexpected behavior in the Rust port, please report them by opening a new issue on the [GitHub Issues](https://github.com/simd-lite/simd-json/issues) page. Make sure to provide detailed information about the bug, including steps to reproduce it.
+
+### Adding Features
+
+If you have ideas for new features or improvements, please discuss  by opening a new issue on the [GitHub Issues](https://github.com/simd-lite/simd-json/issues) section. Engage with the Contributors on new features or improvements and refine your proposal.
+
+### Code Contributions
+
+For code contributions:
+
+1. Fork the project repository on GitHub.
+2. Clone your fork locally: `git clone https://github.com/your-username/your-repo.git`
+3. Create a new branch for your changes: `git checkout -b feature/your-feature`
+4. Write your code, following the Code of Conduct (see [Code of Conduct](#CODE_OF_CONDUCT.md)).
+5. Write unit tests for your code (see [Tests](#tests)).
+6. you can refer to `data`folder for further information (see [data](#data)) 
+7. Push your changes to your fork on GitHub: `git push origin feature/your-feature`
+8. Create a Pull Request (PR) in the project repository, providing a clear description of your changes and linking to any relevant issues or discussions.
+
+## Coding Guidelines
+
+To maintain a consistent and readable codebase, please follow these guidelines:
+
+- Adhere to Rust's official style guide and best practices.
+- Document your code using comments and provide clear explanations for complex logic.
+- Aim for code that is efficient and idiomatic in Rust.
+- Ensure your code is compatible with serde for JSON serialization/deserialization.
+
+## Testing
+
+We require comprehensive test coverage to maintain the quality of the codebase. Write unit tests for your code and ensure that existing tests pass. Use continuous integration tools to automatically run tests for various Rust versions.
+
+## Documentation
+
+Documentation is crucial for helping others understand the project.There are bindings for upstream simdjson available [Here](https://crates.io/crates/simdjson-rust) this includes the README, Dependencies, and any other relevant documentation files.
```

Comment:
The documentation of the wrapper crate doesn't have much value for this section.

I am for linking to alternatives, but that feels more at home in the `README.md` than in the `CONTRIBUTING.md`. And we already [have a link to simdjson-rust](https://github.com/simd-lite/simd-json#other-interesting-things) there.